app/helpers/application_helper.rb

```diff
@@ -5,6 +5,13 @@
 module ApplicationHelper
   include Pagy::Frontend
 
+  def boolean_red_green_tag(bool, text = nil)
+    text ||= bool ? "YES" : "NO"
+    colour = bool ? "green" : "red"
+
+    content_tag(:strong, text, class: "govuk-tag govuk-tag--#{colour}")
```

Comment:
I think I remember @peteryates recommending an alternative for this, some sort of component that comes with a gem.

@peteryates can you remember what that was?

Reply:
Sorry, only just saw this.

https://govuk-components.netlify.app/components/tag/